datasets/it/deputies/crawler.py

```diff
@@ -0,0 +1,147 @@
+from typing import Dict
+
+from zavod import Context, helpers as h
+from zavod.stateful.positions import categorise
+
+# The query retrieves a list of members of
+# the Italian Chamber of Deputies (Camera dei Deputati).
+#
+# Query builder: https://dati.camera.it/sparql
+
+QUERY = """
+SELECT DISTINCT
+  ?persona ?cognome ?nome ?info
+  ?dataNascita ?nato ?luogoNascita ?genere
+  ?inizioMandato ?fineMandato
+  ?nomeGruppo ?sigla ?aggiornamento
+WHERE {{
+  # Person entity (a member of parliament)
+  ?persona ocd:rif_mandatoCamera ?mandato;
+           a foaf:Person.
+
+  # Deputy membership, group affiliation, and legislature
+  ?d a ocd:deputato;
+     ocd:aderisce ?aderisce;
+     ocd:rif_leg <http://dati.camera.it/ocd/legislatura.rdf/repubblica_{legislature}>;
+     ocd:rif_mandatoCamera ?mandato.
+  OPTIONAL {{ ?d dc:description ?info }}
+
+  # Personal information (name, gender, birth details)
+  ?d foaf:surname ?cognome;
+     foaf:gender ?genere;
+     foaf:firstName ?nome.
+
+  OPTIONAL {{
+    ?persona <http://purl.org/vocab/bio/0.1/Birth> ?nascita.
+    ?nascita <http://purl.org/vocab/bio/0.1/date> ?dataNascita;
+              rdfs:label ?nato;
+              ocd:rif_luogo ?luogoNascitaUri.
+    ?luogoNascitaUri dc:title ?luogoNascita.
+  }}
+
+  # System update timestamp (last modified)
+  OPTIONAL {{ ?d <http://lod.xdams.org/ontologies/ods/modified> ?aggiornamento. }}
+
+  # Mandate details (start and end dates)
+  ?mandato ocd:rif_elezione ?elezione.
+  OPTIONAL {{ ?mandato ocd:endDate ?fineMandato. }}
+  OPTIONAL {{ ?mandato ocd:startDate ?inizioMandato. }}
+
+  # Political group affiliation (name and abbreviation)
+  OPTIONAL {{
+    ?aderisce ocd:rif_gruppoParlamentare ?gruppo.
+    ?gruppo <http://purl.org/dc/terms/alternative> ?sigla.
+    ?gruppo dc:title ?nomeGruppo.
+  }}
+
+}}
+"""
+
+
+def build_request_query(legislature: int) -> Dict[str, str]:
+    return {"query": QUERY.format(legislature=legislature), "format": "json"}
+
+
+def crawl_item(context: Context, item):
+    first_name = item.pop("nome").get("value")
+    last_name = item.pop("cognome").get("value")
+    dob = item.pop("dataNascita").get("value")
+    start_date = item.pop("inizioMandato").get("value")
+    end_date = item.pop("fineMandato", {}).get("value")
+
+    entity = context.make("Person")
+    entity.id = context.make_id(first_name, last_name, dob)
+    h.apply_name(entity, first_name=first_name, last_name=last_name)
+    entity.add("sourceUrl", item.pop("persona").get("value"))
+    entity.add("birthPlace", item.pop("luogoNascita").get("value"))
+    entity.add("notes", item.pop("nato").get("value"))
+    entity.add("notes", item.pop("info", {}).get("value"))
+    entity.add("political", item.pop("nomeGruppo").get("value"))
+    entity.add("political", item.pop("sigla").get("value"))
```

Comment:
sigla translates to abbreviation. Is there anything interesting in there? Or is it just an abbreviated version of nomeGruppo?